armi/reactor/assemblies.py

```diff
@@ -1317,6 +1317,37 @@ def getSymmetryFactor(self):
         """Return the symmetry factor of this assembly."""
         return self[0].getSymmetryFactor()
 
+    def rotate(self, deg):
+        # note this can only rotate in 60 degree increments.
```

Comment:
this public method needs a valid docstring with a 1 line summary and then more details describing what it does and how.

Reply:
I have added docstrings for all the functions added.